ChangeLog.d/9126.txt

```diff
@@ -0,0 +1,5 @@
+Default behavior changes
+   * In a PSA-client-only build (i.e. MBEDTLS_PSA_CRYPTO_CLIENT &&
```

Comment:
Question: since such builds are not officially supported yet, isn't there a risk that this changelog entry gives the (wrong) impression that they are officially supported already?

Reply:
I think that's ok: we document `MBEDTLS_PSA_CRYPTO_CLIENT` as experimental, and we don't say much about what it does.